OPCB/economy/tierList_INF.sqf

```diff
@@ -54,24 +57,25 @@ OPCB_econ_TierList_INF = [
 	[
 	"C_VAN_01_TRANSPORT_F",
 	"C_VAN_02_TRANSPORT_F",
-	"C_TRUCK_02_TRANSPORT_F"
+	"C_TRUCK_02_TRANSPORT_F",
+	"C_TRUCK_02_BOX_F",
+	"C_TRUCK_02_FUEL_F"
 	],
 
 	//TIER 9
 	[
 	"RHSUSF_M998_D_2DR_HALFTOP",
 	"RHSUSF_M1151_M240_V2_USARMY_D",
-	"RHSUSF_M1230A1_USARMY_D"
```

Comment:
is this the medical vehicle thingy? I think this would be a good addition to the spawner